.mailmap

```diff
@@ -0,0 +1,12 @@
+Mohsen Azimi <me@azimi.me> Mohsen Azimi <mazimi@apigee.com>
+Mohsen Azimi <me@azimi.me> Mohsen Azimi <msn.azi@gmail.com>
```

Comment:
Please remove my name and email. I get enough emails for Swagger stuff already! 😁